docs/contributor/05-10-e2e_tests.md

```diff
@@ -162,19 +169,52 @@ The test executes the following steps:
 3. Run the test scenario:
 
     ```bash
-    make skr-binding-test
+    make skr-binding
+    ```
+
+## Provisioning Service Tests
+
+### Usage
+
+The test executes the following steps:
+
+1. Sends a call to Provisioning API to provision a Kyma runtime. The test waits until the environment is created.
+2. Creates a Kyma Binding.
+3. Fetches the `sap-btp-manager` secret using kubeconfig from created Kyma Binding.
+4. Fetches a created Kyma Biding.
+5. Deletes created Kyma Binding.
+6. Tries to fetch a `sap-btp-manager` secret using invalidated kubeconfig.
+7. Tries to fetch a deleted Kyma Binding.
```

Comment:
Yes, in step 7 test tries to fetch Kyma Binding which was deleted in step 5. The test should not be able to fetch this Kyma Binding, because it no longer exists.